application/uitests/pom.xml

```diff
@@ -88,6 +88,16 @@
 			<artifactId>flightrecorder.rules.jdk</artifactId>
 			<version>${jmc.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.openjdk.jmc</groupId>
+			<artifactId>flightrecorder.serializers</artifactId>
```

Comment:
Is there anything in the uitests that use serializers and writer?

Reply:
Writer, probably not right now, but doesn't hurt. I think the serializers are used by the Websocket stuff, so added them for good measure. Was trying to figure out why it's not possible to sometimes run the UI-tests after having run the RCP app from within Eclipse. I think the conclusion was that this doesn't matter, but having all of core available shouldn't hurt.